src/cli/commands/version.js

```diff
@@ -118,9 +118,7 @@ export async function setVersion(
   // check if committing the new version to git is overriden
   if (!flags.gitTagVersion || !config.getOption('version-git-tag')) {
     // Don't tag the version in Git
-    return function(): Promise<void> {
-      return Promise.resolve();
-    };
+    return Promise.resolve.bind(Promise);
```

Comment:
If you want to use a single line statement, I'd prefer we use `() => Promise.resolve()` instead of `bind`, I find it more easily parsable by my brain and it has the added benefit that it can have a lower runtime overhead.